include/bluetooth/services/ddfs.h

```diff
@@ -0,0 +1,261 @@
+	 *
+	 * @param mode New ranging mode for distanance measurement.
+	 * @retval 0 If the operation was successful.
+	 *         Otherwise, a negative error code is returned.
+	 */
+	int (*dm_ranging_mode_set)(uint8_t mode);
+
+	/** @brief Indicates that the configuration for Distance Measurement is read.
+	 *
+	 * @param config Pointer to configuration object.
+	 * @retval 0 If the operation was successful.
+	 *         Otherwise, a negative error code is returned.
+	 */
+	int (*dm_config_read)(struct bt_ddfs_dm_config *config);
+
+	/** @brief Azimuth Measurement state callback.
+	 *
+	 * Indicate the CCCD descriptor status of the Azimuth Measurement characteristic.
+	 *
+	 * @param enabled Azimuth Measurement notification status.
+	 */
+	void (*am_notification_config_changed)(bool enebled);
+
+	/** @brief Distance Measurement state callback.
+	 *
+	 * Indicate the CCCD descriptor status of the Distance Measurement characteristic.
+	 *
+	 * @param enabled Distance Measurement notification status.
+	 */
+	void (*dm_notification_config_changed)(bool enebled);
+
+	/** @brief Elevation Measurement state callback.
+	 *
+	 * Indicate the CCCD descriptor status of the Elevation Measurement characteristic.
+	 *
+	 * @param enabled Elevation Measurement notification status.
+	 */
+	void (*em_notification_config_changed)(bool enebled);
+};
+
+/** @brief Structure of distance measurement results. */
+struct bt_ddfs_distance_measurement {
+	/* Quality indicator */
+	enum bt_ddfs_quality quality;
+
+	/* Bluetooth LE Device Address */
+	bt_addr_le_t bt_addr;
+
+	/* Mode used for ranging */
+	enum bt_ddfs_dm_ranging_mode ranging_mode;
+	union {
+		struct _mcpd {
+			/* MCPD: Distance estimate based on IFFT of spectrum */
+			uint16_t ifft;
+
+			/* MCPD: Distance estimate based on average phase slope estimation */
+			uint16_t phase_slope;
+
+			/* RSSI: Distance estimate based on Friis path loss formula */
+			uint16_t rssi_openspace;
+
+			/* Best effort distance estimate */
+			uint16_t best;
+		} mcpd;
+		struct _rtt {
+			/* RTT: Distance estimate based on RTT measurement */
+			uint16_t rtt;
+		} rtt;
+	} dist_estimates;
+};
+
+/** @brief Structure of azimuth measurement results. */
+struct bt_ddfs_azimuth_measurement {
+	/* Quality indicator */
+	enum bt_ddfs_quality quality;
+
+	/* Bluetooth LE Device Address */
+	bt_addr_le_t bt_addr;
+
+	/* Azimuth estimate */
+	uint16_t value;
+};
+
+/** @brief Structure of elevation measurement results. */
+struct bt_ddfs_elevation_measurement {
+	/* Quality indicator */
+	enum bt_ddfs_quality quality;
+
+	/* Bluetooth LE Device Address */
+	bt_addr_le_t bt_addr;
+
+	/* Elevation estimate */
+	int8_t value;
+};
+
+/** @brief Direction and Distance Finding Service initialization parameters. */
+struct bt_ddfs_init_params {
+	/** Initial value for features. */
+	struct bt_ddfs_features dm_features;
+
+	/** Callback functions structure.*/
+	const struct bt_ddfs_cb *cp_cb;
```

Comment:
I think that the cp_ prefix can be removed since these are no longer control point only callbacks.